project-reports/2023/2023-Q1-Hyperledger-Besu.md

```diff
@@ -0,0 +1,56 @@
+</mark>
+
+Besu is having a big boost in activity on a number of metrics. Besu is climbing back its network share on Ethereum Mainnet and hovering around 10%. We now have new contributors from Web3Labs taking on issues and providing private network support. This adds another organization alongside existing contributors. The Discord is very active and monitored by a number of teams. Contributor calls are productive and grants are being distributed equitably. A series of Besu related workshops should also bring in new contributors working alongside the TOC and HLF leadership. We hope to onboard at least one of the banks or FinTechs we chatted with.
+
+Additionally, with Shanghai enabling Ether withdrawals from validators, HLF can begin to use some of these funds to pay for project infrastructure and hopefully some incentives for community engagement down the line alongside CSI and other maintainer groups. 
+
+# Required Information
+All completed for Besu. 
+
+- Have you switched from master to main in all your repos?
+- Have you implemented the [Common Repository Structure](../guidelines/repository-structure.md) in all your repos?
+- Has your project implemented these inclusive language changes listed below to your repo? You can optionally [use the DCI Lint tool](https://github.com/petermetz/gh-action-dci-lint#usage) to make this a recurring action on your repo.
+  - master → main
+  - slave → replicas
+  - blacklist → denylist
+  - whitelist → allowlist
+- Have you added an [Inclusive Language Statement](https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example) to your project's documentation and/or Wiki pages?
+
+# Questions/Issues for the TSC
+
+Will the TSC deliver a unified CI/CD strategy across HLF projects to avoid ambiguity going forward? 
+
+# Releases
+
+Besu was late in releasing its Quarter 1 CalVer quartlerly (23.1.0), due to huge changes to the codebase and a few bugs with Bonsai specifically. The group released 3 updates in this quarter, 23.1.0, 23.1.1, and 23.1.2. 23.1.2 contains the Bonsai refactor and Shanghai hard-fork support for Ethereum Mainnet. 
+
+# Overall Activity in the Past Quarter
+
+Discord is very active as users look for support with both private and public networks. Consensys has worked with new partners to hand over the bulk of private network support but all maintainers are popping in and out to provide valuable support. Some users still ask questions via issue submissions in Git and the mailing list and these are low in volume compared to Discord. We have seen new contirubitons from some external contributors and an uptick of folks dipping into the #besu-contributors channel to look for insight on the codebase. We hope this will continue as we gain traction with contributors. 
```

Comment:
```suggestion
Discord is very active as users look for support with both private and public networks. Consensys has worked with new partners to hand over the bulk of private network support but all maintainers are popping in and out to provide valuable support. Some users still ask questions via issue submissions in Git and the mailing list and these are low in volume compared to Discord. We have seen new contributions from some external contributors and an uptick of folks dipping into the #besu-contributors channel to look for insight on the codebase. We hope this will continue as we gain traction with contributors. 
```